src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,21 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    CATEGORY: {
```

Comment:
- [ ] Kindly define the action type in the correct way. [Link>>>](https://github.com/erikras/ducks-modular-redux#:~:text=//%20Actions%0Aconst%20LOAD%20%20%20%3D%20%27my%2Dapp/widgets/LOAD%27%3B%0Aconst%20CREATE%20%3D%20%27my%2Dapp/widgets/CREATE%27%3B%0Aconst%20UPDATE%20%3D%20%27my%2Dapp/widgets/UPDATE%27%3B%0Aconst%20REMOVE%20%3D%20%27my%2Dapp/widgets/REMOVE%27%3B)

Reply:
Thanks for the Link @Rustamxon7 . I have exhausted it extensively and it is really enlightening. Put I must point out that this is more of a classic React Redux material which creates the reducers, actions and dispatch by hand. However React Redux strongly recommends that the new way of handling redux is by using the RTK- Redux Toolkit.  [See Redux Documentation Link](https://redux.js.org/introduction/getting-started) and [React Redux Toolkit Link](https://react-redux.js.org/introduction/getting-started#an-existing-react-app). 

Again, i have exhausted extensively from the material given during the week on how to create and use Redux firsthand but, the task, required that we use the toolkit with the use of redux hooks such as useSelectors and useDispatch. 

Thanks again, i really appreciate

Reply:
Aha, I got this. Since this time we were not getting used to Redux Toolkit. We mostly created apps with [Redux doc](https://github.com/erikras/ducks-modular-redux). So, I think you have the right not to do the required changes at that stage. Let's change requirements to optional at this poing.